index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8"> 
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
It is great that you give a title to your page but there should be some modification needed. Please give the title according to the page. As the design of the page is a survey form and the heading is also a "A survey form" so it is better that the title of the page should be according to this.